infra/modules/aurora-db/main.tf

```diff
@@ -0,0 +1,201 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+
+###########################
+## Database Configuration ##
+###########################
+resource "aws_rds_cluster" "postgresql" {
+  # checkov:skip=CKV2_AWS_27:have concerns about sensitive data in logs; want better way to get this information
+  # checkov:skip=CKV2_AWS_8:TODO add backup selection plan using tags
+  cluster_identifier = var.service_name
+  engine             = "aurora-postgresql"
+  engine_mode        = "provisioned"
+  database_name      = replace("${var.service_name}", "-", "_")
```

Comment:
nit: could we define a local variable called database_name that has this expression and then use that variable here? that way we can see quickly up top all the generated values and how they are generated.